packages/react-components/react-link/src/components/Link/useLinkStyles.ts

```diff
@@ -99,15 +99,17 @@ const useStyles = makeStyles({
 
 export const useLinkStyles_unstable = (state: LinkState): LinkState => {
   const styles = useStyles();
+  const { appearance, inline, root } = state;
+
   state.root.className = mergeClasses(
     linkClassNames.root,
     styles.root,
     styles.focusIndicator,
-    state.root.as === 'a' && state.root.href && styles.href,
-    state.appearance === 'subtle' && styles.subtle,
-    state.inline && styles.inline,
-    state.appearance === 'subtle' && state.inline && styles.inlineSubtle,
-    state.root['aria-disabled'] && styles.disabled,
+    root.as === 'a' && root.href && styles.href,
+    appearance === 'subtle' && styles.subtle,
+    inline && styles.inline,
+    appearance === 'subtle' && inline && styles.inlineSubtle,
+    root['aria-disabled'] && styles.disabled,
```

Comment:
Would it make sense to have a single `state.disabled = props.disabled || props.disabledFocusable`? It seems like we don't need to differentiate between disabled and disabledFocusable on `state`.

Reply:
We cannot, we're setting `state.disabled` as `disabled && !disabledFocusable` when the link renders as a button so the native `disabled` attribute in the button tag is set correctly.

Reply:
This is updated now.